rmgpy/rmg/reactors.py

```diff
@@ -412,23 +426,35 @@ def to_rms(obj, species_names=None, rms_species_list=None):
         coeffs = obj.coeffs.value_si.tolist()
         return rms.Chebyshev(coeffs, Tmin, Tmax, Pmin, Pmax)
     elif isinstance(obj, ThirdBody):
-        arr = to_rms(obj.arrheniusLow)
-        efficiencies = {spc.label: float(val) for spc, val in obj.efficiencies.items() if val != 1}
-        return rms.ThirdBody(arr, nameefficiencies=efficiencies)
+        arrstr = arrhenius_to_julia_string(obj.arrheniusLow)
+        efficiencies = {species_names[i] : float(val) for i, val in enumerate(obj.get_effective_collider_efficiencies(rmg_species)) if val != 1}
```

Comment:
Are we sure that `species_names` is in the same order as `rmg_species`?  For a phase, it seems like they should be, not sure if we are also using this for interphase.